gen_cert.py

```diff
@@ -229,7 +214,13 @@ def interstitial_factory():
             self.long_course = long_course or cert_data.get('LONG_COURSE', '').encode('utf-8')
             self.issued_date = issued_date or cert_data.get('ISSUED_DATE', '').encode('utf-8') or 'ROLLING'
             self.interstitial_texts = collections.defaultdict(interstitial_factory())
-            self.interstitial_texts.update(cert_data.get('interstitial', {}))
+            interstitial_dict = {
+                key.encode('utf8'): value.encode('utf8')
+                for key, value in cert_data.get('interstitial', {}).items()
+            }
+            self.interstitial_texts.update(interstitial_dict)
+            self.locale = cert_data.get('LOCALE', settings.DEFAULT_LOCALE).encode('utf-8')
```

Comment:
Is there any reason why `LOCALE` key is in all caps? I know we have one or two old keys like that, but most everything else is lowercase, which seems more fitting here. 

`DEFAULT_LOCALE` makes sense because it's a constant.